helm-charts/fluent-bit_values.yaml

```diff
@@ -1,272 +1,228 @@
-    time_key: "@timestamp"
-    # Optional username credential for Elastic X-Pack access
-    http_user:
-    # Password for user defined in HTTP_User
-    http_passwd:
-    # Optional TLS encryption to ElasticSearch instance
-    tls: "off"
-    tls_verify: "off"
-    # TLS certificate for the Elastic (in PEM format). Use if tls=on and tls_verify=on.
-    tls_ca: ""
-    # TLS debugging levels = 1-4
-    tls_debug: 1
-  stackdriver: {}
-
-  ##
-  ## Ref: http://fluentbit.io/documentation/current/output/http.html
-  ##
-  http:
-    host: 127.0.0.1
-    port: 80
-    uri: "/"
-    http_user:
-    http_passwd:
-    tls: "off"
-    tls_verify: "on"
-    tls_debug: 1
-    ## Specify the data format to be used in the HTTP request body
-    ## Can be either 'msgpack' or 'json'
-    format: msgpack
-    headers: []
-
-parsers:
+serviceMonitor:
   enabled: false
-  ## List the respective parsers in key: value format per entry
-  ## Regex required fields are name and regex. JSON and Logfmt required field
-  ## is name.
-  regex: []
-  logfmt: []
-  ##  json parser config can be defined by providing an extraEntries field.
-  ##  The following entry:
-  ## json:
-  ##   - extraEntries: |
-  ##       Decode_Field_As  escaped log do_next
-  ##       Decode_Field_As  json log
-  ##
-  ##  translates into
-  ##
-  ##   Command       |  Decoder  | Field | Optional Action   |
-  ##   ==============|===========|=======|===================|
-  ##   Decode_Field_As  escaped   log  do_next
-  ##   Decode_Field_As  json log
-  ##
-  json: []
+  # namespace: monitoring
+  # interval: 10s
+  # scrapeTimeout: 10s
+  # selector:
+  #  prometheus: my-prometheus
 
-env: []
+prometheusRule:
+  enabled: false
+  # namespace: ""
+  # additionnalLabels: {}
+  # rules:
+  # - alert: NoOutputBytesProcessed
+  #   expr: rate(fluentbit_output_proc_bytes_total[5m]) == 0
+  #   annotations:
+  #     message: |
+  #       Fluent Bit instance {{ $labels.instance }}'s output plugin {{ $labels.name }} has not processed any
+  #       bytes for at least 15 minutes.
+  #     summary: No Output Bytes Processed
+  #   for: 15m
+  #   labels:
+  #     severity: critical
+
+dashboards:
+  enabled: false
+  labelKey: grafana_dashboard
+  annotations: {}
 
-## Annotations to add to the DaemonSet's Pods
-podAnnotations: {}
+livenessProbe: {}
 
-## By default there different 'files' provides in the config
-## (fluent-bit.conf, custom_parsers.conf). This defeats
-## changing a configmap (since it uses subPath). If this
-## variable is set, the user is assumed to have provided,
-## in 'existingConfigMap' the entire config (etc/*) of fluent-bit,
-## parsers and system config. In this case, no subPath is
-## used
-fullConfigMap: false
-
-## ConfigMap override where fullname is {{.Release.Name}}-{{.Values.existingConfigMap}}
-## Defining existingConfigMap will cause templates/config.yaml
-## to NOT generate a ConfigMap resource
-##
-existingConfigMap: ""
+readinessProbe: {}
 
+resources:
```

Comment:
@angeloskyratzakos Can we set some limits and requests here? We should do the same in the gitlab utility values.